DemoApp/BannerView.swift

```diff
@@ -0,0 +1,47 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChatUI
+import UIKit
+
+final class BannerView: UIView {
+    private let label = UILabel()
+    
+    override func didMoveToSuperview() {
+        super.didMoveToSuperview()
+        setUpLayout()
+        defaultAppearance()
+    }
+    
+    func update(text: String) {
+        label.text = text
+    }
+    
+    private func setUpLayout() {
+        guard let superview = superview else { return }
+        
+        addSubview(label)
+        
+        translatesAutoresizingMaskIntoConstraints = false
```

Comment:
The view's superview should decide about this, not the view itself.

Reply:
Reworked